snuba/datasets/storages/groupassignees.py

```diff
@@ -61,16 +43,13 @@ def get_bulk_loader(self, source: BulkLoadSource, dest_table: str) -> SingleTabl
     storage_key=StorageKey.GROUPASSIGNEES,
     storage_set_key=StorageSetKey.EVENTS,
     schemas=StorageSchemas(read_schema=schema, write_schema=schema),
-    table_writer=GroupAssigneeTableWriter(
-        write_schema=schema,
-        stream_loader=KafkaStreamLoader(
-            processor=GroupAssigneeProcessor(POSTGRES_TABLE),
-            default_topic="cdc",
-            pre_filter=CdcTableNameMessageFilter(POSTGRES_TABLE),
-        ),
-        postgres_table=POSTGRES_TABLE,
+    stream_loader=KafkaStreamLoader(
```

Comment:
same nit: could you please write the argument in the same order as they are in the parent class? Now they are many, it makes things easier to follow.